sys/linux/dev_usbmon.txt

```diff
@@ -0,0 +1,88 @@
+# Copyright 2017 syzkaller project authors. All rights reserved.
+# Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+# https://elixir.bootlin.com/linux/latest/source/Documentation/usb/usbmon.txt
+# https://elixir.bootlin.com/linux/latest/source/drivers/usb/mon/mon_bin.c
+
+include <linux/fcntl.h>
+include <linux/ioctl.h>
+
+define MON_IOC_MAGIC	0x92
```

Comment:
Please add a brief comment saying why we are defining all these things here rather than include kernel header.